simphony/cuds/material.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from simphony.core.data_container import DataContainer
+
+
+class Material(object):
+    """A class that represents a material.
+
+    Class describes a material and its data.
+
+    Attributes
+    ----------
+    description : str
+        textual description of the material
+    uid : uuid.UUID
+        the id of the material. If None, then it is generated
+    data : DataContainer
+        data associated with this material
+
+    """
+    def __init__(self, uid=None, data=None, description=""):
```

Comment:
I think that SSB expects that user can save a name parameter to the material with
```material_foo = Material(name="foo")```
(see #227)
Maybe there should be a name argument together with (optional) description. @ahashibon, could you please clarify this. In my opinion using the name would be sufficient in most cases, and the description could be used to add more detailed information about the material.